commercetools-java-client-core/src/main/java/io/sphere/sdk/client/UserAgentUtils.java

```diff
@@ -16,12 +16,15 @@
     private UserAgentUtils() {
     }
 
+    public static String unknownUserAgent = "commercetools-java-v1/unknown";
+    public static String userAgent = "commercetools-sdk-java-v1";
```

Comment:
both properties can be made final and non public

Reply:
yes I noticed it I committed it ;)